fiftyone/utils/eval/activitynet.py

```diff
@@ -0,0 +1,592 @@
+                mrec = np.hstack([[0], rec, [1]])
+                idx = np.where(mrec[1::] != mrec[0:-1])[0] + 1
+                ap = np.sum((mrec[idx] - mrec[idx - 1]) * mprec[idx])
+
+                # Interpolate precision values for PR curve plotting purposes
+                inds = np.searchsorted(rec, recall, side="left")
+                try:
+                    for ri, pi in enumerate(inds):
+                        q[ri] = pre[pi]
+                except:
+                    pass
+
+                precision[iou_threshs.index(t)][classes.index(c)] = q
+                classwise_AP[iou_threshs.index(t)][classes.index(c)] = ap
+
+        return ActivityNetDetectionResults(
+            matches,
+            precision,
+            recall,
+            classwise_AP,
+            iou_threshs,
+            classes,
+            eval_key=eval_key,
+            gt_field=gt_field,
+            pred_field=pred_field,
+            missing=missing,
+            samples=samples,
+        )
+
+
+class ActivityNetDetectionResults(DetectionResults):
+    """Class that stores the results of a ActivityNet detection evaluation.
+
+    Args:
+        matches: a list of
+            ``(gt_label, pred_label, iou, pred_confidence, gt_id, pred_id)``
+            matches. Either label can be ``None`` to indicate an unmatched
+            segment
+        precision: an array of precision values of shape
+            ``num_iou_threshs x num_classes x num_recall``
+        recall: an array of recall values
+        classwise_AP: an array of average precision values of shape
+            ``num_iou_threshs x num_classes``
+        iou_threshs: the list of IoU thresholds
+        classes: the list of possible classes
+        eval_key (None): the evaluation key for this evaluation
+        gt_field (None): the name of the ground truth field
+        pred_field (None): the name of the predictions field
+        missing (None): a missing label string. Any unmatched segments are
+            given this label for evaluation purposes
+        samples (None): the :class:`fiftyone.core.collections.SampleCollection`
+            for which the results were computed
+    """
+
+    def __init__(
+        self,
+        matches,
+        precision,
+        recall,
+        classwise_AP,
+        iou_threshs,
+        classes,
+        eval_key=None,
+        gt_field=None,
+        pred_field=None,
+        missing=None,
+        samples=None,
+    ):
+        super().__init__(
+            matches,
+            eval_key=eval_key,
+            gt_field=gt_field,
+            pred_field=pred_field,
+            classes=classes,
+            missing=missing,
+            samples=samples,
+        )
+        self.precision = np.asarray(precision)
+        self.recall = np.asarray(recall)
+        self.iou_threshs = np.asarray(iou_threshs)
+        self._classwise_AP = classwise_AP.mean(0)
+
+    def plot_pr_curves(self, classes=None, backend="plotly", **kwargs):
```

Comment:
https://github.com/voxel51/fiftyone/pull/1490 added two new things to this method for COCO evaluation
- Added an `iou_thresh` parameter to plot a specific threshold's curves
- Passed confidence thresholds through to the plotting routine (for tooltips)

Can you add the same features here please?